### PR TITLE
Stop flickering + added loader in AddressSelector

### LINKED ISSUE #4103: stop flickering in address selector
Right now, the list of completions gets cleared once the address selector starts loading again. This creates an annoying flickering effect and prevents the user from selecting a value when the next completion lookup is already running.

Introduce some way to let the user know the selector is lookup up completions, but don't clear the completions immediately.

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes #4103

### CODE REVIEW

File: js/src/ui/Form/AddressSelect/addressSelect.js [hunk 1]
```diff
@@ -182,17 +183,20 @@ class AddressSelect extends Component {
...
+        <div className={ styles.outerInput }>
+          <input
+            id={ id }
+            className={ styles.input }
+            placeholder={ ilHint }
 
-          onBlur={ this.handleInputBlur }
-          onFocus={ this.handleInputFocus }
-          onChange={ this.handleChange }
+            onBlur={ this.handleInputBlur }
```
Comment (jacogr) [inline js/src/ui/Form/AddressSelect/addressSelect.js]: Alphabetical order is fine, just aligning style everywhere, no extra breaks needed. (I would rather not introduce more styling preferences yet again, props is still all-over the place.)

File: js/src/ui/Form/AddressSelect/addressSelect.js [hunk 2]
```diff
@@ -182,17 +183,20 @@ class AddressSelect extends Component {
...
+            onFocus={ this.handleInputFocus }
+            onChange={ this.handleChange }
 
-          ref={ this.setInputRef }
-        />
+            ref={ this.setInputRef }
```
Comment (jacogr) [inline js/src/ui/Form/AddressSelect/addressSelect.js]: Same here, extra "it looks pretty to some" break.

File: js/src/ui/Loading/loading.js [hunk 3]
```diff
@@ -21,15 +21,22 @@ import styles from './loading.css';
...
 
+  static defaultProps = {
+    className: '',
+    size: 2
+  };
+
   render () {
-    const size = (this.props.size || 2) * 60;
+    const { className, size } = this.props;
+    const _size = size * 60;
```
Comment (jacogr) [inline js/src/ui/Loading/loading.js]: Prefer to keep `_name` to function params where supplied. So may make more sense as `calculatedSize`.

File: js/src/ui/Form/AddressSelect/addressSelect.js [hunk 4]
```diff
@@ -210,6 +214,19 @@ class AddressSelect extends Component {
...
 
+  renderLoader () {
+    if (!this.store.loading) {
+      return null;
+    }
+
+    return (
+      <Loading
+        className={ styles.loader }
+        size={ 0.5 }
```
Comment (jacogr) [inline js/src/ui/Form/AddressSelect/addressSelect.js]: Do we need to override the defaults? How does this work for consistency across the full UI?

File: js/src/ui/Form/InputAddress/inputAddress.js [hunk 5]
```diff
@@ -76,7 +76,7 @@ class InputAddress extends Component {
 
     const props = {};
 
-    if (!readOnly && !disabled) {
+    if (!disabled) {
```
Comment (jacogr) [inline js/src/ui/Form/InputAddress/inputAddress.js]: Why was this changed? The whole readOnly/disabled is an issue overall for this component, seems that was added due to some issue somewhere and now removed due to some issue here.

Comment (jacogr) [inline js/src/ui/Form/InputAddress/inputAddress.js]: Yes, added here for contract params - https://github.com/ethcore/parity/commit/d16ab5eac50963c0ef9fb5bfa9e33a75f68cf914

**Comment by jacogr** [comment]
1. Accounts column has a horizontal scrollbar, seems like extra space ended up in that column. It is the first time I'm noticing this, possibly not introduced here, but breaks the layout.

2. Headings are significantly larger than modal headings, should really be the same. (Very visible not with the 75% overlay - same would go for input as well). Once again not quite new.

3. Padding around modal top is not the same as on the sides, should be aligned. (If one is bigger it is typically top/bottom not left/right, this is reversed)

(Mentioning all of these here since this does fix dialog issues and would like to get it right so we can move on)

File: js/src/ui/Form/AddressSelect/addressSelect.js (hunk 4 above)
Comment (ngotchac) [inline js/src/ui/Form/AddressSelect/addressSelect.js]: Really really big. This size fills perfectly the input

File: js/src/ui/Form/InputAddress/inputAddress.js (hunk 5 above)
Comment (ngotchac) [inline js/src/ui/Form/InputAddress/inputAddress.js]: Just tested, fine as it is. It wasn't properly adding the right underline line on focus, and would break the tabbing.

**Comment by ngotchac** [comment]
1. Not sure that I can reproduce it. Could reproduce a horizontal scroll bar on very very narrow screen. Fixed it though.

Fixed to other comments.

**Comment by jacogr** [comment]
1. Weird. Shows up for me each time. Will re-look to see if it still happens with the current version.

File: js/src/ui/Form/AddressSelect/addressSelect.js [hunk 6]
```diff
@@ -25,6 +25,7 @@ import TextFieldUnderline from 'material-ui/TextField/TextFieldUnderline';
 
 import AccountCard from '~/ui/AccountCard';
 import InputAddress from '~/ui/Form/InputAddress';
+import Loading from '~/ui/Loading';
```
Comment (jacogr) [inline js/src/ui/Form/AddressSelect/addressSelect.js]: Probably makes sense as `import { Loading, Protal } from 'ui'` now

**Comment by derhuerst** [comment]
@jacogr

1. cannot replicate
2. definitely agree, but wouldn't necessarily change it now. seems out of scope of this pr.

**Comment by ngotchac** [comment]
@derhuerst 2. Has been fixed ;)

**Comment by jacogr** [comment]
1 is due to this https://github.com/ethcore/parity/issues/4160
